docs/sql-ref-syntax-ddl-drop-table.md

```diff
@@ -19,4 +19,70 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+
+Deletes the table and removes the directory associated with this table from the file system
+if this is not `EXTERNAL` table. If the table is not present it throws exception.
+
+In case of External table it only deletes the metadata from metastore database and it will not remove the directory 
```

Comment:
Nit: In case of External table, only the associated metadata information are removed from the metastore database.